### PR TITLE
Hotfix/clickable video player

### PR DESCRIPTION
**Pull request by IvanSkadorva**

### CODE REVIEW

**Comment by kuzkov** [comment]
I don't think that using native progress element is a good idea. Custom implementation of progress bar could be more flexible and customizable than progress element.

**Comment by kuzkov** [comment]
image
Media queries can be nested

File: src/app/pages/video-player/progress-bar/progress-bar.component.scss [hunk 1]
```diff
@@ -13,13 +12,23 @@
...
-    font-family: Roboto, "Arial Narrow", Arial sans-serif;
+    font-family: Roboto, 'Arial Narrow', Arial sans-serif;
+  }
+}
+
+@media (max-width: 1919px) {
+  .progress-bar__timeline {
+    width: 84%;
   }
 }
```
Comment (kuzkov) [inline src/app/pages/video-player/progress-bar/progress-bar.component.scss:34]: Media queries can be nested

**Review by kuzkov — CHANGES_REQUESTED** [review]
image
I don't think that's good)))

File: src/app/pages/video-player/progress-bar/progress-bar.component.scss (hunk 1 above)
Comment (IvanSkadorva) [reply src/app/pages/video-player/progress-bar/progress-bar.component.scss:34]: fixed

**Pull request merged**